aikau/src/test/resources/alfresco/navigation/PathTreeTest.js

```diff
@@ -187,6 +187,33 @@ define(["intern!object",
                   });
          },
 
+         // See AKU-770 - As with previous two tests, but in this case checking at the root node which wasn't previously working
+         "Check root node refresh on folder add": function() {
+            return browser.findById("ADD_FOLDER_AT_ROOT_label")
+                  .clearXhrLog()
+                  .click()
+               .end()
+
+               .getLastXhr()
+                  .then(function(xhr) {
+                     /* jshint maxlen:500*/
+                     assert.include(xhr.request.url, "/slingshot/doclib/treenode/node/alfresco/company/home/documentLibrary/?perms=false&children=false&max=500&libraryRoot=workspace%3A%2F%2FSpacesStore%2Fb4cff62a-664d-4d45-9302-98723eac1319", "Node refresh not requested");
+                  });
+         },
+
+         "Check root node refresh on folder delete": function() {
+            return browser.findById("DELETE_FOLDER_AT_ROOT_label")
+                  .clearXhrLog()
+                  .click()
+               .end()
+
+               .getLastXhr()
+                  .then(function(xhr) {
+                     /* jshint maxlen:500*/
+                     assert.include(xhr.request.url, "/slingshot/doclib/treenode/node/alfresco/company/home/?perms=false&children=false&max=500&libraryRoot=workspace%3A%2F%2FSpacesStore%2Fb4cff62a-664d-4d45-9302-98723eac1319", "Node refresh not requested");
```

Comment:
Not sure the error message is necessary here. There's a single assert and the name of the test should be enough to know what's failing.